source/advanced-installation.txt

```diff
@@ -170,7 +170,7 @@ Install the MongoDB C Driver
 
 The mongocxx driver builds on top of the `MongoDB C driver <https://www.mongodb.com/docs/drivers/c/>`__.
 
-The build of mongocxx-3.9.0 automatically downloads and installs the C driver if the
+The build of mongocxx-{+full-version+} automatically downloads and installs the C driver if the
```

Comment:
Q: since this is a variable, just confirming this will be true for v4.1 and all future versions?